spec/integration/user_show_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe 'User Index Page', type: :feature do
```

Comment:
- [ ] Hi, Your test failed, can you work on it again!

![Capture](https://user-images.githubusercontent.com/63972670/220869382-87e80292-20b5-4679-b1a9-a67db47628d3.JPG)